In d face match

gowthamd3 opened the pull request:
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [ ] I attest that the connector works and I verified by deploying and testing all the operations.
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

sriyen-msft commented on apiProperties.json:
```diff
@@ -0,0 +1,23 @@
+{
```
@gowthamd3 Please remove this file

sriyen-msft commented on certified-connectors/IN-D Aadhaar Number Masking/README.md:
```diff
@@ -0,0 +1,20 @@
+# IN-D Aadhaar Number Masking
```
@gowthamd3 Please remove all the files added for `IN-D Aadhaar Number Masking` from this PR. You can raise a separate PR for this connector, so that it would be easy for us to review and track.

sriyen-msft commented:
@gowthamd3 Also, this PR has branch conflicts, please update your local branch with remote branch before pushing any commit for the PR.

sriyen-msft commented:
@gowthamd3 Please remove the entire folder called "IN-D Insurance (ICD10 & CPT)" from the repo as it is duplicate to "IN-D Insurance"

gowthamd3 commented:
> @gowthamd3 Please remove the entire folder called "IN-D Insurance (ICD10 & CPT)" from the repo as it is duplicate to "IN-D Insurance"

Hi @sriyen-msft i have removed insurance file from this PR also removed folder, still  i can see deleted file with empty json data in this PR?, will this be an issue to Merge PR?. if yes help me out to resolve the issue.

sriyen-msft commented:
@gowthamd3 I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

sriyen-msft commented:
@gowthamd3 Since there are no updates for this PR, I am closing out and you can reopen it whenever you or your team is ready for pushing updates.
